### DISCUSSION TITLE
Pywall integration

### DISCUSSION THREAD

**Discussion by antonioszm** [Shina-SG/Shina-Fox#46] [Ideas]
The idea is to integrate the color palette generated by pywal into firefox Shina, pywalfox already exists but using this theme it only changes the icons at the top and the sidebar, the integration would be in the parts where pywalfox does not change the color like the part that separates the website from the url

**Comment by antonioszm** [comment]
I implemented it myself and it was really easy
just import at the beginning of the code
example:` @import url('file:///home/antonioszm/.cache/wal/colors.css');
`
and use `var(--background)` and etc. the result is really good

Before I had accidentally imported it wrong and it simply resulted in the most beautiful browser I've ever seen, so I continued with the wrong way lol

**Reply by Izurai** [reply]
Can you please give more details about what you've done to implement pywal ? Many thanks.

**Reply by antonioszm** [reply]
In global settings I import my colors.css from pywal cache
![image](https://github.com/user-attachments/assets/51227395-7919-4e4f-88a1-d19198c8835e)

Now in the code I change the colors to my variables (imported from cache file)
![image](https://github.com/user-attachments/assets/bbe5614d-3ffb-487a-97b2-c4d3906d018a)

Tip: Just change in the theme that you are using, exemple: Im using the dark theme so i only modify the dark variables
